### PR TITLE
With 81xx WiFi driver

### PR DESCRIPTION
**Pull request by alfert**
This is a configuration to add support for the popular Realtek 81xx WiFi adapters, usually as a USB dongle. 

It requires changes in both, the linux kernel and the base packages. The kernel driver is compiled as module, otherwise the kernel compilation fails. What does not work is the use of the Nerves_interim_wifi package. However, as @amolenaar showed (https://github.com/amolenaar/elixir-brewberry-pi), if you run `modprobe` and `wpa_supplicant` directly, `nerves_networking` is enough. See also https://github.com/alfert/led_nerves for another example of this approach. The next step is to extend `nerves_networking` to setup the WiFi adapters properly. 

@amolenaar did the research for this approach. He should get the honors.

### CODE REVIEW

File: linux-4.4.defconfig [hunk 1]
```diff
@@ -78,7 +78,6 @@ CONFIG_SYN_COOKIES=y
 # CONFIG_IPV6 is not set
 CONFIG_CFG80211=y
 CONFIG_CFG80211_INTERNAL_REGDB=y
-CONFIG_CFG80211_WEXT=y
```
Comment (amolenaar) [inline linux-4.4.defconfig]: I'm not sure whether removing this option will inhibit other wifi device drivers from working.

**Comment by GregMefford** [comment]
@alfert and @amolenaar: Thank you both for your effort on figuring this out! 🚀 🎈 
I'm planning to spend some time on improving the WiFi support and do some more testing to validate that these changes are compatible with the other supported chipsets (which I also need to list clearly in the documentation 😅).

File: linux-4.4.defconfig (hunk 1 above)
Comment (fhunleth) [inline linux-4.4.defconfig]: I know that it affects a wifi driver on a different platform. I'm not sure if there's a common wifi driver used on the RPi that requires wext. It seems like including wext shouldn't be an issue for 81xx support.

**Comment by fhunleth** [comment]
@alfert Could you squash related commits together? I.e., run `git rebase -i origin/master` and squash the `silly me...` commit with a previous one so that it's not in the long term history of the project.

**Comment by alfert** [comment]
@fhunleth Yes, I will do it. I will also add the wext-config again. It was not really a conscious decision to  remove that option.

**Comment by alfert** [comment]
Done, learned a lot about squashing to open pull requests in the last hour. Hope it went well.

File: linux-4.4.defconfig (hunk 1 above)
Comment (fhunleth) [inline linux-4.4.defconfig]: It turns out that the RTL8192CU wifi driver requires wext so this option is pulled in automatically. In other words, it's no longer needed in the defconfig. See:
https://github.com/raspberrypi/linux/blob/rpi-4.4.y/drivers/net/wireless/realtek/rtl8192cu/Kconfig#L4

**Comment by fhunleth** [comment]
Looks good to me. I'll leave it open a little longer in case others have comments, but if not, I'll merge this evening. Thanks!

**Comment by GregMefford** [comment]
@fhunleth: I assume we need to also make this change on the other platforms for consistency, right? I don't have time tonight, but I was planning to try it out on each of the RPi variants since I have them all and a relevant dongle (I think).

**Comment by fhunleth** [comment]
@GregMefford Making the change for the RPi2 would be nice. No need to change the RPi3, since it has built-in WiFi.

**Pull request merged**